### ISSUE TITLE
Should F61 also trigger 2.2.1, 2.2.4 as well as 3.2.5 ?

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1183]
F41 fails Success Criterion 2.2.1, 2.2.4, and 3.2.5 due to using meta refresh to reload the page
https://www.w3.org/WAI/WCAG21/Techniques/failures/F41.html

F61 deals with a very similar situation, and the F61 examples and test procedure seem like they should also fail 2.2.1, 2.2.4
https://www.w3.org/WAI/WCAG21/Techniques/failures/F61.html

**Comment by patrickhlauke** [comment]
+1

**Comment by JAWS-test** [comment]
I do not think that F41 and F61 are the same. If I understand it correctly, F41 describes a case where the whole page is updated, while F61 only updates parts of the page. In particular, Example 3 for F61 describes a case of a page update that was triggered by the user and therefore does not violate 2.2.1. If Example 3 were removed at F61, I could imagine that 2.2.1 and 2.2.4 could also be included at F61.
